app/controllers/course/condition/levels_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Course::Condition::LevelsController < Course::ConditionsController
+  load_resource :level_condition, class: Course::Condition::Level.name, parent: false
+
+  def new
+  end
+
+  def create
+    @level_condition.acting_as.conditional = @conditional
+    @level_condition.acting_as.course = @course
```

Comment:
Maybe this can be set as a `before_action`. Controller methods should be very very short.